src/scanoss/scanner.py

```diff
@@ -983,16 +965,23 @@ def wfp_folder(self, scan_dir: str, wfp_file: str = None):
             raise Exception(f'ERROR: Please specify a folder to fingerprint')
         if not os.path.exists(scan_dir) or not os.path.isdir(scan_dir):
             raise Exception(f'ERROR: Specified folder does not exist or is not a folder: {scan_dir}')
+        file_filters = FileFilters(debug=self.debug, trace=self.trace, quiet=self.quiet,
+                                   scanoss_settings=self.scan_settings,
+                                   all_extensions=self.all_extensions,
+                                   all_folders=self.all_folders,
+                                   hidden_files_folders=self.hidden_files_folders,
+                                   skip_size=self.skip_size,
+                                   skip_folders=self.skip_folders,
+                                   skip_extensions=self.skip_extensions,
+                                   operation_type='scanning'
```

Comment:
you're passing here operation_type = 'scanning', should be 'fingerprinting'

also i think is cleaner just to instantiate it once and pass operation type as argument to the function